scripts/RT/weps2sweep_worker.py

```
@@ -0,0 +1,366 @@
+            "-i3",  # temp debuggin
+            "-I2",  # Run the given management cycles, TODO
+            "-n0",  # Don't create new input files
+            f"-o{payload.dt:%d%m%Y}",  # Generate SWEEP input on this date
+            f"-P{tmpdir}",  # Path to files
+            "-t0",  # No confidence interval reported
+            "-T0",  # No deep furrow effect
+            "-W1",  # simple runoff method, perf
+            "-u0",  # No resurfacing of buried roots, perf opt?
+        ]
+        try:
+            proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)